frontend/test/metabase/scenarios/admin/people/people.cy.spec.js

```diff
@@ -88,6 +88,23 @@ describe("scenarios > admin > people", () => {
       cy.findByText(FULL_NAME);
     });
 
+    it("should allow admin to create new users without first name or last name (metabase#22754)", () => {
+      const { email } = TEST_USER;
+      cy.visit("/admin/people");
+      clickButton("Invite someone");
+
+      // bit of a hack since there are multiple "Email" nodes
+      cy.findByLabelText("Email").type(email);
+      clickButton("Create");
```

Comment:
`cy.findByLabelText("Email")` doesn't look like a hack at all to me. I'd actually prefer it exclusively for selecting inputs in Cypress. Lets us be sure the forms are more accessible

Reply:
To be honest. The whole test is duplicated from another test including that comment. The only part that I change is when inviting someone, I don't type the first name or last name. 😅